src/regions/de/i18n/steps.de.ts

```diff
@@ -22,56 +22,56 @@ export default {
   'askForLocationOptions-ST': 'Sachsen-Anhalt',
   'askForLocationOptions-SH': 'Schleswig-Holstein',
   'askForLocationOptions-TH': 'Thüringen',
-  'askHasHadContactOptions-Option1': 'Ja, einiges davon trifft auf mich zu.',
+  'askHasHadContactOptions-Option1': 'Ja, davon trifft etwas auf mich zu',
   'askHasImmuneDecreasedOptions-Option1':
-    'Ja, einiges davon trifft auf mich zu.',
-  'askHasImmuneDecreased2Options-Option1': 'Ja, das bin ich.',
-  'askHasImmuneDecreasedOptions-Option2': 'Nein, nichts trifft auf mich zu.',
+    'Ja, davon trifft etwas auf mich zu',
+  'askHasImmuneDecreased2Options-Option1': 'Ja, das tue ich',
+  'askHasImmuneDecreasedOptions-Option2': 'Nein, davon trifft nichts zu',
   yes: 'Ja',
   no: 'Nein',
   askForLocation:
-    'In welcher Provinz oder welchem Gebiet befinden Sie sich derzeit?',
+    'In welchem Bundesland befinden Sie sich gerade?',
   askForLocationInfo:
-    'Ich werde in der Lage sein, die Informationen so zuzuschneiden, dass sie für Ihre Region gelten.',
+    'Mit Hilfe Ihrer Angabe kann ich die wichtigsten Informationen für die ausgewählte Region anzeigen.',
   askHasHadContact: `
-  Hatten Sie **naher Kontakt** mit einem bestätigten oder wahrscheinlichen Fall von COVID-19?`,
+  Hatten Sie **engen Kontakt** mit einer Person, die ein bestätigter oder wahrscheinlicher Fall von COVID-19 ist?,
```

Comment:
Missing closing of string

```suggestion
  Hatten Sie **engen Kontakt** mit einer Person, die ein bestätigter oder wahrscheinlicher Fall von COVID-19 ist?`,
```